composer.json

```diff
@@ -18,9 +18,9 @@
         }
     ],
     "require": {
-        "php": ">=5.3.2",
-        "symfony/framework-bundle": "~2.7",
-        "symfony/form": "~2.7",
+        "php": ">=5.5.0",
```

Comment:
Symfony `2.8` still [has php `5.3.9` as a minimum](https://github.com/symfony/symfony/blob/2.8/composer.json), and `3.0` [has `5.5.9` as minimum](https://github.com/symfony/symfony/blob/3.0/composer.json).

This means the bundle won't follow the Symfony deprecation process, which is fine with me. We also already started using the FQCN for the formtypes. But think you should at least raise php to `5.5.9` then.